modules/grafana/files/dashboards/sidekiq.json

```diff
@@ -71,20 +80,20 @@
           "steppedLine": false,
           "targets": [
             {
-              "refId": "A",
-              "target": "aliasByNode(averageSeriesWithWildcards(stats.gauges.govuk.app.$Application.*.workers.queues.$Queues.enqueued, max, 5), 7)",
+              "hide": false,
+              "refId": "B",
+              "target": "aliasByNode(averageSeriesWithWildcards(stats.gauges.govuk.app.$Application.*.workers.queues.$Queues.enqueued, 5), 4, 7)",
```

Comment:
The title of this panel was changed from "Queue Length" to "Jobs Processed", but this Graphite query will get the enqueued jobs, not a rate.